tests/queries/0_stateless/02884_create_view_with_sql_security_option.reference

```diff
@@ -28,6 +28,7 @@ OK
 100
 100
 OK
+OK
```

Comment:
It's a bit difficult to read this because this 'OK' actually means 'syntax error'.